src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,264 @@
+expected_output = {
+    "cts_device_identity": "SJC-ab-gw1",
+    "cts_sgt_caching": "Disabled",
+    "cts_ingress_sgt_caching": "Disabled",
+    "cts_sg_epg_translation": "Disabled",
+    "interfaces_in_manual_mode": 0,
+    "interfaces_in_ifc_states": {
+        "init": 0,
+        "authenticating": 0,
+        "authorizing": 0,
+        "sap_negotiating": 0,
+        "open": 0,
+        "held": 0,
+        "disconnecting": 0,
+        "invalid": 0
+    },
+    "cts_events_statistics": {
+        "authentication_success": 0,
+        "authentication_reject": 0,
+        "authentication_failure": 0,
+        "authentication_logoff": 0,
+        "authentication_no_resp": 0,
+        "authorization_success": 0,
+        "authorization_failure": 0,
+        "sap_success": 0,
+        "sap_failure": 0,
+        "port_auth_fail": 0
+    },
+    "installed_list": {
+        "name": "CTSServerList1-0089",
+        "count": 7,
+        "server_ip": {
+            "10.100.123.1": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.2": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.3": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.4": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.5": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.6": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
```

Comment:
these times, better to have the unit in key name and value is just integer. Please change all where applicable.
```
'idle_time_mins': 60,
'deadtime_secs': 20,
```

Reply:
Updated